### DISCUSSION TITLE
litematica

### DISCUSSION THREAD

**Discussion by Matifey** [maruohon/litematica#550] [Q&A]
ran into a problem, the mod does not work, installed fabric and malilib, when I pick up the wand, the menu appears but does not scroll and the pick block hotkeys switch appears on the M button. help me please

**Comment by maruohon** [comment]
Which keyboard layout (QWERTY, AZERTY) are you using? Check your Hotkeys config category, have you changed some of the keys such as `operationModeChangeModifier` (which is the modifier key you need to hold while scrolling with the mouse to change the "tool mode")? You can see what keys have been modified if the `RESET` button is enabled.

Also a common issue with the tool functions not working is if you accidentally toggle off the tool with `M + T`, but if you still see the tool HUD on the bottom left when holding the item, then it's not disabled in your case (unless you are in the Schematic Edit mode, since the tool HUD always shows as a "warning" when in that mode).
